src/server/auth/server/api_server.go

```diff
@@ -168,10 +168,15 @@ func (a *apiServer) EnvBootstrap(ctx context.Context) error {
 		// handle oidc clients & this cluster's auth config
 		if a.env.Config.AuthConfig != "" && a.env.Config.IdentityClients != "" {
 			var config auth.OIDCConfig
+			var pachdClientSecret string
 			var clients []identity.OIDCClient
 			if err := yaml.Unmarshal([]byte(a.env.Config.AuthConfig), &config); err != nil {
 				return errors.Wrapf(err, "unmarshal auth config: %q", a.env.Config.AuthConfig)
 			}
+			if err := yaml.Unmarshal([]byte(a.env.Config.AuthClientSecret), &pachdClientSecret); err != nil {
```

Comment:
Is this YAML or just a raw value?  If YAML, we might need a test for the case where it's unset (dunno how well the empty string unmarshals as YAML).